package.json

```diff
@@ -3,8 +3,5 @@
   "version": "1.4.0",
   "peerDependencies": {
     "eslint": ">=3.14.1"
-  },
-  "dependencies": {
-    "tslint-eslint-rules": "^4.1.1"
```

Comment:
what about these things that were playing out?

```
+      "eslint-plugin-node": "^11.1.0",
+      "eslint-plugin-promise": "^5.1.0",
+      "eslint-plugin-standard": "^5.0.0",
+      "eslint-plugin-jsdoc": "^35.4.2",
+      "eslint-plugin-prefer-arrow": "^1.2.3",
```

were those all because of tslint-eslint?

Reply:
`prefer-arrow` is a legit (hidden) dependency. Those first three - I don't know why you needed them, but you shouldn't. I didn't need them when I pulled this into Audiate. JsDoc I removed here and is no longer needed.